client/gameObjects/Skeleton.js

```diff
@@ -105,13 +105,40 @@ export default class Skeleton extends Phaser.GameObjects.Image {
 
     this.scene = scene;
 
+    this.range = [];
+
     this.label = this.scene.add
       .text(this.x, this.y, this.name)
       .setOrigin(0.5, -1.0)
       .setPosition(this.x, this.y - this.displayHeight / 2);
     this.label.depth = this.depth;
   }
 
+  setAlphaRange(alpha = 1) {
+    this.range.forEach((tile) => {
+      tile.setAlpha(alpha);
+    });
+  }
+
+  showRange() {
```

Comment:
It's getting tight in `Skeleton.js`, we should think about extracting UI / tile effects into separate file.